ratlib/timeutil.py

```diff
@@ -105,3 +108,10 @@ def format_timestamp(ts):
     if isinstance(ts, datetime.date):
         return ts.strftime("%b %d, %Y")
     return ts.strftime("%H:%M:%S")
+
+def utc_now_tz()->str:
+    """
+
+    :return: current UTC time in a timezone-aware string.
+    """
+    return datetime.datetime.now(datetime.timezone.utc).isoformat().replace('+00:00', 'Z')
```

Comment:
This seems to break the display on DWB

Reply:
Which format exactly does the Dispatch Board expect? The timestamp this produces is a valid ISO 8601 Timestamp.
Though i am curious why the +0:00 is replaced with a Z - it's both valid.

Reply:
dispatch board expected a timestamp without a timezone because I added the timezone to workaround the bug this change fixed. No further changes to mecha need to be made, I just need to remove the workaround.

Also, we replace +00:00 with a Z for consistency with timestamps provided by the api.